CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ Bugfixes:
 
 Other improvements:
 
-## [Unreleased] - YEAR-MONTH-DATE
+## [v5.0.0](https://github.com/purescript/purescript-prelude/releases/tag/v5.0.0) - 2021-02-07
```

Comment:
This step may need some manual intervention, as some libraries didn’t include “YEAR-MONTH-DATE”, some didn’t add the changes in the unreleased section but rather in a new section with the proposed new tag...so just something we’ll have to be careful about.

Reply:
I have three different strategies for the prelude, the node libraries and then all the others libraries 😅